### PR TITLE
Update to client 2.0.0

### LINKED ISSUE #22: Update to support client version 2.0
With https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/2.0.0, this should be update.

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #22

### CODE REVIEW

**Comment by SeanFeldman** [comment]
@Azure/azure-service-bus-write please review

**Comment by nemakam** [comment]
>   </ItemGroup>

not required

---
Refers to: src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj:36 in 4cd4b18. [](commit_id = 4cd4b18d9abbfa7fc40e85f714bdbc264cd07c43, deletion_comment = False)

**Review by nemakam — APPROVED** [review]
@SeanFeldman , looks good. I've given a minor comment. Once you take care of that, i'll merge the changes.

**Comment by SeanFeldman** [comment]
@nemakam I have to admit, looking at the GH comment, I can't figure out what you were referring to 🙂 
image

File: src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj [hunk 1]
```diff
@@ -29,19 +26,9 @@
...
-    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
-  </PropertyGroup>
-  
+ 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="1.*" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="2.*" />
   </ItemGroup>
   
   <ItemGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
```
Comment (nemakam) [inline src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj]: You could remove this `ItemGroup` as well

Comment (nemakam) [reply src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj]: This is the update to the `not required` comment.
Looks like some bug in the PR tool

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus.MessageIdPlugin/Microsoft.Azure.ServiceBus.MessageIdPlugin.csproj]: done

**Review by nemakam — APPROVED** [review]

**Comment by SeanFeldman** [comment]
This was merged into `dev`, but was never released 🙁 
The current version on nuget is a preview, with dependency on Microsoft.Azure.ServiceBus `0.0.6-preview`. Please address it.

**Pull request merged**